docs/configuration.rst

```diff
@@ -0,0 +1,92 @@
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| rubix.enable.file.staleness-check        | Check file metadata for updates from remote filesystem when true.      | boolean           | true             | S             |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| rubix.stale.fileinfo.expiry.period       | (**rubix.enable.file.staleness-check** must be enabled)                |                   |                  |               |
+|                                          | The time file metadata will be cached before expiring.                 | integer (s)       | 36000            | S             |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| rubix.parallel.warmup                    | Warmup data asynchronously when true.                                  | boolean           | false            | C & S         |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+
+Network
+-------
+
++------------------------------------------+-------------------------------------------------------------------------------+-------------------+------------------+---------------+
+| Option                                   | Description                                                                   | Type              | Default          | Client/Server |
++==========================================+===============================================================================+===================+==================+===============+
+| hadoop.cache.data.bookkeeper.port        | The port on which the BookKeeper server is listening.                         | integer           | 8899             | C & S         |
++------------------------------------------+-------------------------------------------------------------------------------+-------------------+------------------+---------------+
+| hadoop.cache.data.local.server.port      | The port on which the Local Data Transfer server is listening.                | integer           | 8898             | C             |
++------------------------------------------+-------------------------------------------------------------------------------+-------------------+------------------+---------------+
+| hadoop.cache.data.client.num-retries     | The maximum number of retry attempts to connect to the BookKeeper server.     | integer           | 3                | C & S         |
```

Comment:
The maximum number of retry attempts to execute any Bookkeeper thrift server function